packages/api/src/aggregate/StringFilter.ts

```diff
@@ -17,11 +17,11 @@
 import type { BaseFilterOptions, CatchThemAll } from "./BaseFilter.js";
 import type { Just } from "./Just.js";
 
-interface StringFilterOptions extends BaseFilterOptions<string> {
+export interface StringFilterOptions extends BaseFilterOptions<string> {
   "$startsWith": string;
   "$containsAllTermsInOrder": string;
-  "$containsAnyTerm": string;
-  "$containsAllTerms": string;
+  "$containsAnyTerm": string | { term: string; fuzzySearch?: boolean };
```

Comment:
Can we expose somewhere what fuzzySearch means exactly? Not sure if dev console docs is best or if we need inline.

Reply:
Also, removing export for now